tests/cases/compiler/reachabilityChecks9.ts

```diff
@@ -0,0 +1,28 @@
+// @strict: true
+// @noEmit: true
```

Comment:
There are no errors for these test cases - do you have to disable `allowUnreachableCode` or something?